tests/unit/dataloader/test_tf_dataloader.py

```diff
@@ -98,63 +98,6 @@ def test_simple_model():
     _ = model.evaluate(loader)
 
 
-def test_nested_list():
```

Comment:
This PR doesn't change the output representation. the output types and shapes should be the same as before. Apart from the torch version which now outputs the offsets with the last value correctly and consistently returns the scalar columns as 2-d arrays with the last dim of size 1

The reason for the removal of the test is mostly because it is now broken due to the difference between our implementation of the [pull_apart_list ](https://github.com/NVIDIA-Merlin/dataloader/blob/v0.0.4/merlin/dataloader/loader_base.py#L575) behaving in a different way between cudf/pandas. I'll try restoring these two lines https://github.com/NVIDIA-Merlin/dataloader/blob/v0.0.4/merlin/dataloader/loader_base.py#L577-L579 and see If I can restore this test.

The current implementation and what this test is checking isn't really nested list support. It currently loses the information about the nesting during the transformation. And what you get out of the dataloader cannot be turned back into the nested lists in the original data.

Reply:
Updated to restore this test `test_nested_list`, and tests are all passing. We can leave this to another PR if we want to change the behaviour. Thanks for checking about this.

Reply:
Thanks for the explanation.